### PR TITLE
ENH: covariates collection

### LINKED ISSUE #249: Add temperature, humidity and atmospheric pressure measurement in SOP
Add details on how to measure the temperature, humidity and atmospheric pressure (fixed by #234) 
Add picture of the thermometer and barometer (see #257)

### PR DESCRIPTION
**Pull request by celprov**
ENH: integrate procedure to collect covariates in the main protocol
enh: add details on how to measure certain covariates
wip: it is not yet clear how to measure all covariates

edit: closes #249

### CODE REVIEW

**Comment by celprov** [comment]
@acionca I started to define how to measure the covariates. Feel free to build on top of this PR ;)

@esavary @oesteban @acionca before merging this, you might want to define on which computer you will collect the covariates as well as the session notes and replace the `[ON WHICH COMPUTER SHOULD THIS BE DONE?]` place holders.

File: docs/data-collection/pre-session.md [hunk 1]
```diff
@@ -52,9 +52,43 @@ Instructions of operations to be performed before the participant arrival, **bef
 
     ![nuprep-tube](https://shop.neurospec.com/media/catalog/product/cache/8af976a03d45f9b6d7f91e69feeeffb9/w/e/weaver_nuprep-skin-prep-gel-single-tube-front.jpg)
 
+- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session [ON WHICH COMPUTER SHOULD THIS BE DONE?]
```
Comment (acionca) [inline docs/data-collection/pre-session.md]: Fixed by #219 
```suggestion
- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session (a laptop should be available to fill the issue).
```

File: docs/data-collection/pre-session.md [hunk 2]
```diff
@@ -52,9 +52,43 @@ Instructions of operations to be performed before the participant arrival, **bef
...
+    - [ ] Under the section `Issues`, click on <span class="consolebutton green">New issue</span>
+    - [ ] This should lead you a page that looks like this 
+        ![](../assets/images/session_issue_template.png)
+        - [ ] Click on <span class="consolebutton green">Get started</span> on the issue template `Scan session`.
+    - [ ] Modify the title of the issue by replacing `yyy` with the session index. If you don't remember the session index of today, check the session index of the last issue.
 - [ ] Verify that your phone is on ringing mode so the participants can reach you
 - [ ] Check the time regularly to be on time to meet with the participant at the predefined location
 
+## Collection of covariates
+- [ ] Open a new issue in [the github repository dedicated to collecting covariates](https://github.com/TheAxonLab/hcph-mood-quest/) [ON WHICH COMPUTER SHOULD THIS BE DONE?]
```
Comment (acionca) [inline docs/data-collection/pre-session.md]: Same, fixed by #219 
```suggestion
- [ ] Open a new issue in [the github repository dedicated to collecting covariates](https://github.com/TheAxonLab/hcph-mood-quest/) (a laptop should be available to fill the issue).
```

**Review by acionca — APPROVED** [review]
Looks good to me, I don't see any other instructions that we could add - the survey is straightforward. I believe instruction to measure humidity and temperature could be added on a separate PR).

I believe there are some redundancies with other PR (such as #219) so we might be carefull when merging. Thanks for putting this together !

File: docs/data-collection/pre-session.md [hunk 3]
```diff
@@ -49,15 +49,61 @@ Instructions of operations to be performed before the participant arrival, **bef
 
     ![nuprep-tube](https://shop.neurospec.com/media/catalog/product/cache/8af976a03d45f9b6d7f91e69feeeffb9/w/e/weaver_nuprep-skin-prep-gel-single-tube-front.jpg)
 
-- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session [ON WHICH COMPUTER SHOULD THIS BE DONE?]
+- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session (a laptop should be available to fill the issue).
     - [ ] Under the section `Issues`, click on <span class="consolebutton green">New issue</span>
-    - [ ] This should lead you to a page that looks like this 
+    - [ ] This should lead you a page that looks like this
```
Comment (acionca) [inline docs/data-collection/pre-session.md]:
```suggestion
    - [ ] This should lead you to a page that looks like this 
```

**Review by acionca — APPROVED** [review]
Hey @celprov, sorry I worked a bit on your fork/branch so that I could fix some conflicts with the mkdocs branch (from the merge of #219). I think it is ready to be merged - feel free to add any modification/suggestion and/or merge when you feel like it's ready (the only thing left would be related to #258 but it could be added later).

**Comment by celprov** [comment]
Cool :)
Let's merge this now and work on #258 in another PR.

PS the schedule in the SOP is a good idea!

**Pull request merged**